src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/evosuite/validation/ValidationToolWindowFactory.kt

```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.evosuite.validation
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.ToolWindow
+import com.intellij.openapi.wm.ToolWindowFactory
+
+/**
+ * This class is responsible for creating the tabs and the UI of the TestGenie tool window.
+ */
+class ValidationToolWindowFactory : ToolWindowFactory {
```

Comment:
Should we possibly use this also for displaying build and evosuite output as well?

Reply:
That would be a nice feature.